.github/workflows/integration.yml

```diff
@@ -60,3 +60,10 @@ jobs:
         with:
           fail_ci_if_error: false
           token: ${{ secrets.CODECOV_TOKEN }}
+      - name: Upload test results to Codecov
+        if: ${{ github.event_name == 'schedule' || (github.event_name == 'push') || github.event_name == 'workflow_dispatch'}}
+        uses: codecov/test-results-action@v1
+        with:
+          fail_ci_if_error: false
+          files: ./target/surefire-reports/TEST*
```

Comment:
Folks, sorry for the late moment but aren't we going to miss the failsafe-reports this way?

```yaml
./target/surefire-reports/TEST*
./target/failsafe-reports/TEST*
```

Reply:
@tishun Do we use the failsafe plugin in Jedis? I might have missed it.